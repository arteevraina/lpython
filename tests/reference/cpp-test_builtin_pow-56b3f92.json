{
    "basename": "cpp-test_builtin_pow-56b3f92",
    "cmd": "lpython --no-color --show-cpp {infile}",
    "infile": "tests/../integration_tests/test_builtin_pow.py",
    "infile_hash": "b7d1d5e1592f5078961eb228c756e424d394f5f0383a1577f1cced1b",
    "outfile": null,
    "outfile_hash": null,
    "stdout": null,
    "stdout_hash": null,
    "stderr": "cpp-test_builtin_pow-56b3f92.stderr",
    "stderr_hash": "6935bb3dd01caf67154e6dee0bb6480b2d3ecb270bc5038444391acc",
    "returncode": 2
}