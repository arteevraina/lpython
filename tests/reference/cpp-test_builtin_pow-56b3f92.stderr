warning: The module 'lpython_builtin' located in $DIR/src/bin/../runtime/lpython_builtin.py cannot be loaded
  --> tests/../integration_tests/test_builtin_pow.py:11:16
   |
11 |     assert i32(pow(a, b)) == 32
   |                ^^^^^^^^^ imported here

semantic error: The symbol '_lpython_floordiv' not found in the module 'lpython_builtin'
   --> $DIR/src/bin/../runtime/lpython_builtin.py:209:15
    |
209 |     if (n_ - (n_ // 2)*2) == 0:
    |               ^^^^^^^ 
